dashboard/app/views/reports/pd_progress.html.haml

```diff
@@ -10,5 +10,11 @@
   var headers = #{ headers.try(:to_json)};
   var ht = new Handsontable(document.getElementById('progress'), {
       data: data,
-      colHeaders: headers
+      colHeaders: headers,
+      cells: function(row, col, prop) {
+        if([12, 19, 26, 34, 37, 45, 52, 59, 63, 73, 77, 81, 82].indexOf(col) != -1) {
```

Comment:
Can you forsee other situations in which we want to directly interact with or identify if a level is a PD Progress page? 

Also, are the these IDs guaranteed by the codebase, or are they incidental (I know, for example, that level IDs can be different between different instances of the site)?